src/System.Globalization/tests/System.Globalization.Tests.csproj

```diff
@@ -124,6 +125,9 @@
     <Compile Include="$(CommonTestPath)\System\PlatformDetection.cs">
       <Link>Common\System\PlatformDetection.cs</Link>
     </Compile>
+    <Compile Include="$(CommonTestPath)\System\PlatformDetection.Unix.cs" Condition="'$(TargetGroup)' != 'uap'">
```

Comment:
If more start using it then we should make a Platform.Detection.Windows.cs that just returns false, so we don't need any #if's.